README.md

```diff
@@ -83,10 +83,26 @@ and [Report Transaction](https://dev.maxmind.com/minfraud/report-transaction/) A
 
 ### minFraud API ###
 
-To use the minFraud API, create a new `\MaxMind\MinFraud` object. The constructor
-takes your MaxMind account ID, license key, and an optional options array as
-arguments. This object is immutable. You then build up the request using the
-`->with*` methods as shown below. Each method call returns a new object. The
+To use the MinFraud API, you'll need to create a new `MaxMind\MinFraud` object. The constructor
```

Comment:
Why'd we lose the `\`? Can we keep this as `minFraud API`? The original phrasing seems fine.